rpcserver.go

```diff
@@ -2327,11 +2342,18 @@ func (s *rpcServer) prepareLeasesResponse(ctx context.Context,
 					clearingPrice = batch.ClearingPrices[duration]
 				}
 
+				premiumAmt := chanAmt
+				auctionType := ourOrder.Details().AuctionType
+				if auctionType == order.BTCOutboundLiquidity {
+					premiumAmt += btcutil.Amount(
```

Comment:
Hmm, so the _buyer_ pays the _seller_ a premium based on the amount of BTC the _buyer_ commits? 

I'd say this is probably the biggest open question I have (and I think my main source of confusion with this concept). One key question I have is: as an asker, why is someone _paying_ me to accept a channel?

Reply:
Yes, the idea is that I have a node well positioned in the network ( my outbound liquidity is valued by someone else) and they want to open a channel to me so they can route through me.

What I am selling is my outbound liquidity, the amount that you will be able to route through me. So that's what I price (your self balance amt).

We can already see many nodes in the network that do not accept channels by default so nobody messes up with their liquidity balance flows, here node operators will be able to price that, so buyer/seller have a new market where both can benefit without having to use the defensive strategy of "If you want to open a channel to us send us an email"

By now, as an asker,  I can set the min chan amount but eventually I should also be able to set a max amt so I can limit how much of my outbound liquidity I want to put in the market.

Reply:
>as an asker, why is someone paying me to accept a channel?
you have a scarce good, confirmed channel coins and user/LSP wants to zero conf route through the seller